src/prefect/server/schemas/actions.py

```diff
@@ -876,7 +883,10 @@ class WorkPoolUpdate(ActionBaseModel):
     is_paused: Optional[bool] = Field(default=None)
     base_job_template: Optional[Dict[str, Any]] = Field(default=None)
     concurrency_limit: Optional[NonNegativeInteger] = Field(default=None)
-
+    storage_configuration: schemas.core.WorkPoolStorageConfiguration = Field(
+        default_factory=schemas.core.WorkPoolStorageConfiguration,
+        description="The storage configuration for the work pool.",
+    )
```

Comment:
Should this default to `None`?

Reply:
ah yes probably, i think we should update [this](https://github.com/PrefectHQ/nebula/pull/9462/files#r2004342439) then too?

Reply:
updated in [a9006a0](https://github.com/PrefectHQ/prefect/pull/17527/commits/a9006a0dbb602aca98d204fb65f50e60cd2fc61f)